examples/healthcareapis/healthcaremedtechservice.yaml

```
@@ -0,0 +1,140 @@
+                          "deviceIdExpression": "$.deviceid",
+                          "timestampExpression": "$.measurementdatetime",
+                          "values": [
+                            {
+                              "required": "true",
+                              "valueExpression": "$.heartrate",
+                              "valueName": "hr"
+                            }
+                          ]
+                        }
+                      }
+                    ]
+      }
+    eventhubConsumerGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    eventhubNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    eventhubNamespaceNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    identity:
+    - type: SystemAssigned
+    location: east us
+    workspaceIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+
+apiVersion: healthcareapis.azure.upbound.io/v1beta1
+kind: HealthcareWorkspace
+metadata:
+  annotations:
+    meta.upbound.io/example-id: healthcareapis/v1beta1/healthcaremedtechservice
+  labels:
+    testing.upbound.io/example-name: example
+  name: hcmsexample
+spec:
+  forProvider:
+    location: East US
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+
+apiVersion: azure.upbound.io/v1beta1
+kind: ResourceGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: healthcareapis/v1beta1/healthcaremedtechservice
+  labels:
+    testing.upbound.io/example-name: example
+  name: hcmsexample
+spec:
+  forProvider:
+    location: East US
+
+---
+
+apiVersion: eventhub.azure.upbound.io/v1beta1
+kind: EventHub
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kusto/v1beta1/healthcaremedtechservice
+  labels:
+    testing.upbound.io/example-name: example
+  name: hcmsexample
+spec:
+  forProvider:
+    messageRetention: 1
+    namespaceNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    partitionCount: 1
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+
+apiVersion: eventhub.azure.upbound.io/v1beta1
+kind: ConsumerGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kusto/v1beta1/healthcaremedtechservice
+  labels:
+    testing.upbound.io/example-name: example
+  name: hcmsexample
+spec:
+  forProvider:
+    eventhubNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    namespaceNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+
+apiVersion: eventhub.azure.upbound.io/v1beta1
+kind: EventHubNamespace
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kusto/v1beta1/healthcaremedtechservice
```

```suggestion
    meta.upbound.io/example-id: healthcareapis/v1beta1/healthcaremedtechservice
```

could you please change the group name?